checkov/secrets/consts.py

```diff
@@ -6,6 +6,7 @@ class ValidationStatus(Enum):
     VALID = 'Valid'
     INVALID = 'Invalid'
     UNKNOWN = 'Unknown'
+    UNAVAILABLE = 'UNAVAILABLE'
```

Comment:
@Eliran-Turgeman should be `Unavailable`?

Reply:
Yes!